### ISSUE TITLE
Stuck at "walk to switch"

### ISSUE THREAD

**Issue by ielenia** [Tetraforce/PythonAdventure#1]
When I reach "Walk to switch", the game treats my answer as an invalid response.

**Comment by LankyWhisper624** [comment]
```python
while control:
    print(replies2[0], sep="\t")
    reply = input(query[0]).lower
    if reply == "walk to switch":
        print(strings2[1])
        control = False
    else:
        print(invalid[0])
        control = True
control = True
```
This is the code that controls that. There should be nothing wrong with it. However, I will compare it with the previous question validators to see what's changed.

**Comment by LankyWhisper624** [comment]
```python
strings1 = ["You wake up in a dark and mysterious cave.", "What do you want to do?", "You get up.", "You can't see anything.", "You run into a wall. It feels hard and cold...", "You start to head back when suddenly, you see a fire start.", "You can now see everything in the room."]
strings2 = ["You see a switch on the wall. Next to it is the outline of a door.", "You arrive at the switch", "The door opens.", "You are temporarily blinded by the sun as the door opens.", "You are standing outside a grassy field, and the door closes behind you.", "There is a statue of a bird nearby."]
replies1 = ["Get up", "Look around", "Walk forward"]
replies2 = ["Walk to switch", "Activate switch", "Head outside", "Look around", "Go to bird statue"]
query = ["What do you want to do? ", "What do you want to do now? "]
invalid = ["Please enter a valid response."]
```
Here are all the arrays. ```replies2[0].lower``` is ```"walk to switch"```.  
I don't know why this is happening. It may just be a bug with python.

**Comment by ielenia** [comment]
I believe that parentheses are missing; line 3 should read:
```python
reply = input(query[0]).lower()
```
This is also missing from everything calling from `replies2[]`.

**Comment by LankyWhisper624** [comment]
It should work, but python should return a syntax error...  
Fixing it now.

**Comment by LankyWhisper624** [comment]
Should be fixed. Additional testing may be required.

**Comment by ielenia** [comment]
Fixed with release 0.0.1.
